packages/react-instantsearch/src/connectors/connectRange.js

```diff
@@ -190,8 +190,13 @@ export default createConnector({
   getProvidedProps(props, searchState, searchResults) {
     const { attributeName, precision, min: minBound, max: maxBound } = props;
     const results = getResults(searchResults, this.context);
-    const stats = results ? results.getFacetStats(attributeName) || {} : {};
-    const count = results
+    const isFacetExist = results && results.getFacetByName(attributeName);
```

Comment:
`facetExists` seems like a better name

Reply:
I prefer to prefix all the boolean value like one with `is` or `has` it make them explicit. Would you agree for `isFacetExists` ?

Reply:
makes sense, but grammatically it reads weird. What about `hasFacet` as something which matches the pattern but also grammar?